### PR TITLE
Fixed project issues with the Analyzer Projects

### PR DESCRIPTION
**Pull request by christopheranderson**
Fixes https://github.com/Azure/azure-webjobs-sdk/issues/1501

### CODE REVIEW

File: src/Analyzer/WebJobs.VSAnalyzer.Vsix/Analyzer2.Vsix.csproj [hunk 1]
```diff
@@ -147,5 +147,5 @@
 
 
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
+  <!--<Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />-->
```
Comment (fabiocav) [inline src/Analyzer/WebJobs.VSAnalyzer.Vsix/Analyzer2.Vsix.csproj]: Is this removal required if you build using the VS Developer command prompt?

Comment (christopheranderson) [reply src/Analyzer/WebJobs.VSAnalyzer.Vsix/Analyzer2.Vsix.csproj]: I didn't try the command prompt, but it was failing in VS. The targets file doesn't exist. Do you expect that to change from the VS command prompt?

Comment (fabiocav) [reply src/Analyzer/WebJobs.VSAnalyzer.Vsix/Analyzer2.Vsix.csproj]: Not if the SDK is not installed. This essentially introduces the SDK dependency, which you'd need to install, and at that point, building from the dev command prompt should work. I just wonder if removing this will have some impact on the analyzer projects.

Comment (christopheranderson) [reply src/Analyzer/WebJobs.VSAnalyzer.Vsix/Analyzer2.Vsix.csproj]: I installed the SDK and it still failed. It's fair to say I didn't do any explicit testing the analyzer project, just that it built. Is there a flow to test it I can follow?

**Comment by christopheranderson** [comment]
Opened https://github.com/Azure/azure-webjobs-sdk/issues/1515 to address Fabio's comments about the Analyzer project needing investigation as to whether it is still functional.
